src/main/java/reactor/netty/instrumentation/zipkin/TracingContext.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2011-2019 Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.netty.instrumentation.zipkin;
+
+import brave.Span;
+import com.sun.istack.internal.NotNull;
+import java.util.function.Consumer;
+import reactor.util.annotation.NonNull;
+import reactor.util.annotation.Nullable;
+import reactor.util.context.Context;
+
+/**
+ * Span related methods for {@link Context}.
+ */
+public class TracingContext {
```

Comment:
nit if this is not a user-type, hide it?

Reply:
It's a decorator for reactive `Context` which provides span related methods for user level code.

for example:

```
Mono.subscriberContext()
  .flatMap(ctx -> {
    TracingContext.of(ctx).span(span -> span.annotate("log message"));
    return ...;
  });

Reply:
cool. heh so, I'll nag you then :) javadoc the example? :D

Reply:
I've added Javadoc examples for each user level methods